src/controller/event/handler/replication/replication.go

```diff
@@ -15,13 +15,16 @@
 package replication
 
 import (
+	"context"
+	"fmt"
+	"strconv"
+
 	"github.com/goharbor/harbor/src/controller/event"
+	project2 "github.com/goharbor/harbor/src/controller/project"
```

Comment:
Alias the pkg name to `project2` is not very good please keep it as `project', and we can make the var name `project` to `proj` or `pro' or `p` when needed.